openedx/features/course_experience/views/latest_update.py

```diff
@@ -47,9 +47,4 @@ def latest_update_html(cls, request, course):
         Returns the course's latest update message or None if it doesn't have one.
         """
         # Return the course update with the most recent publish date
-        ordered_updates = get_ordered_updates(request, course)
-        content = None
-        if ordered_updates:
-            content = ordered_updates[0]['content']
-
-        return content
+        return get_current_update_for_user(request, course)
```

Comment:
So this change does mean that the latest_update.py view now respects whether an update has been dismissed. I think that's fine, because if this waffle flag is on, nothing is dismissable and there would be no change. And if a course turns on this flag, I still think we'd want to respect previous dismissals.